prometheus/build.gradle

```diff
@@ -17,7 +22,8 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: "${jackson_version}"
     implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-cbor', version: "${jackson_version}"
     compileOnly group: 'org.opensearch.client', name: 'opensearch-rest-high-level-client', version: "${opensearch_version}"
-    api group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.9.3'
+    implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.9.3'
+    implementation 'com.github.babbel:okhttp-aws-signer:1.0.2'
```

Comment:
is there any AWS official package we can leverage? I am worry about the future CVE maintainance of this package.

Reply:
I couldn't find out an interface for in official library for only SigV4 signing. If we use AWS SDK we might need to use different http request library which will requires lot of changes.
Will create an issue to do POC and get away with this library if required. Also this library internallly uses AWS SDK in some ways.